CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ set(XPP_LIBRARIES
 #
 # TODO drop python2 once ubuntu and debian ship python3-xcbgen in their
 # maintained distros
-foreach(CURRENT_EXECUTABLE python3 python python2 python2.7)
+foreach(CURRENT_EXECUTABLE ${PYTHON_EXECUTABLE} python3 python python2 python2.7)
```

Comment:
According to the documentation of cmake, this variable is deprecated.
https://cmake.org/cmake/help/latest/module/FindPythonInterp.html

We should use `Python3_EXECUTABLE`, see https://cmake.org/cmake/help/latest/module/FindPython3.html#module:FindPython3

Reply:
xpp still has to support cmake 3.0 and cmake 3.0 doesn't have FindPython3, that's why I didn't change it.

Reply:
Oh Indeed.

Reply:
I'm not really sure what the procedure for changing the minimum cmake version is but I would guess any change would qualify as a breaking change